CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Change Log
 
-## Upcoming
+## 1.8 (unreleased)
```

Comment:
1.8.0?

Reply:
i guess we can still add the .0 when we do the release